app/controllers/pagerouters/EligibilityRouter.scala

```diff
@@ -29,17 +29,20 @@ object EligibilityRouter {
       routes.YourBillController.yourBill
     } else {
       EligibilityErrors.toEligibilityError(eligibilityResult.eligibilityRules) match {
-        case Some(MultipleReasons)            => routes.IneligibleController.genericIneligiblePage
-        case None                             => routes.IneligibleController.genericIneligiblePage
-        case Some(HasRlsOnAddress)            => routes.IneligibleController.genericIneligiblePage
-        case Some(MarkedAsInsolvent)          => routes.IneligibleController.genericIneligiblePage
-        case Some(IsLessThanMinDebtAllowance) => routes.IneligibleController.genericIneligiblePage
-        case Some(IsMoreThanMaxDebtAllowance) => routes.IneligibleController.debtTooLargePage
-        case Some(DisallowedChargeLockTypes)  => routes.IneligibleController.genericIneligiblePage
-        case Some(ExistingTtp)                => routes.IneligibleController.alreadyHaveAPaymentPlanPage
-        case Some(ChargesOverMaxDebtAge)      => routes.IneligibleController.debtTooOldPage
-        case Some(IneligibleChargeTypes)      => routes.IneligibleController.genericIneligiblePage
-        case Some(MissingFiledReturns)        => routes.IneligibleController.fileYourReturnPage
+        case Some(MultipleReasons)                   => routes.IneligibleController.genericIneligiblePage
+        case None                                    => routes.IneligibleController.genericIneligiblePage
+        case Some(HasRlsOnAddress)                   => routes.IneligibleController.genericIneligiblePage
+        case Some(MarkedAsInsolvent)                 => routes.IneligibleController.genericIneligiblePage
+        case Some(IsLessThanMinDebtAllowance)        => routes.IneligibleController.genericIneligiblePage
+        case Some(IsMoreThanMaxDebtAllowance)        => routes.IneligibleController.debtTooLargePage
+        case Some(DisallowedChargeLockTypes)         => routes.IneligibleController.genericIneligiblePage
+        case Some(ExistingTtp)                       => routes.IneligibleController.alreadyHaveAPaymentPlanPage
+        case Some(ChargesOverMaxDebtAge)             => routes.IneligibleController.debtTooOldPage
+        case Some(IneligibleChargeTypes)             => routes.IneligibleController.genericIneligiblePage
+        case Some(MissingFiledReturns)               => routes.IneligibleController.fileYourReturnPage
+        //todo someone ask Karen if we will have unique pages for these, if not, delete this todo and leave as generic
```

Comment:
looking at SUPP-571 it does look like those flags do result in a generic ineligibility page if true

Reply:
Ok I'll remove this comment